packages/react-native/src/primitives/PasswordField/PasswordField.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+import { icons } from '../../assets';
+import { IconButton } from '../IconButton';
+import { TextField } from '../TextField';
+import { PasswordFieldProps } from './types';
+
+export default function PasswordField({
+  disabled,
+  hideShowPassword = false,
```

Comment:
Since it is an optional prop it could be undefined, which doesn't really make sense semantically to me, so setting default value to false. Lmk your thoughts.

Reply:
Prefer leaving it `undefined` but do not feel super strongly